proposals/0007-rn-versions-lifecycle-support.md

```diff
@@ -0,0 +1,69 @@
+# RFC: React Native Versions Lifecycle & Support
+
+
+
+>As we are finding ways to better support React Native community, we are moving towards supporting a limited amount of versions of React Native better.
+
```

Comment:
```suggestion
>As we are finding ways to better support React Native community, we are moving towards supporting a limited amount of versions of React Native better.
```

Reply:
Handed in https://github.com/react-native-community/discussions-and-proposals/pull/513